$OpenBSD$
Patch for Gawk 4 has been taken from Gentoo bugracker:
  https://bugs.gentoo.org/show_bug.cgi?id=374143#c9
Planned for inclusion in Virtuoso 6.1.4 release.
--- libsrc/Wi/sql_to_c.awk.orig	Tue Oct 25 02:47:14 2011
+++ libsrc/Wi/sql_to_c.awk	Tue Oct 25 03:12:20 2011
@@ -340,9 +340,18 @@ BEGIN   {
 		    if (length() < 2)
 		      next
 		  }
+
                 # does escape the symbols
 		fun = $0
-	        gsub ( /\\/, "\\\\", fun)
+	        q = "\\\\"
+	        if (PROCINFO["version"] ~ /^4/) {
+                    if (in_xsl_mode == 1)
+                        gsub ( q, q q q q, fun)
+                    else
+                        gsub ( q, q q, fun)
+                } else
+	            gsub ( q, q, fun)
+
 		# remove whitespace except when there is just a semicolon
 		if ((in_xsl_mode == 0) && (in_xsd_mode == 0))
   		  {
